### ISSUE TITLE
Configure sregistry client to use external sregistry hub

### ISSUE THREAD

**Issue by sgabenov** [singularityhub/sregistry#184]
Could you please add documentation notes on how to configure sregistry client to use private sregistry, which run not on a local host, but on dedicated server?

**Comment by vsoch** [comment]
It comes down to user tokens - if the user has permission to see a container collection, he can, otherwise not. Did you see https://singularityhub.github.io/sregistry//setup-teams ?

**Comment by sgabenov** [comment]
Yep, i have reviewed it. But still could not find out, how can i push singularity image to my private hub (for example http://sregistry.scp.xxx.net) with cli. How can i specify where to search for a sregistry from client?

**Comment by vsoch** [comment]
Can you show me the push command that is not working?

**Comment by vsoch** [comment]
(and the error message please)

**Comment by vsoch** [comment]
And in case you haven't seen this page -> https://singularityhub.github.io/sregistry-cli/client-registry

**Comment by vsoch** [comment]
Typically push would be in the format

```
export SREGISTRY_CLIENT=registry
$ sregistry push registry://sregistry.scp.xxx.net/container/repo
```
if you don't have https, then I think you need to also export `SREGISTRY_REGISTRY_NOHTTPS` too.

**Comment by vsoch** [comment]
The easier way is if you have the `sregistry.scp.xxx.net` defined / exported in your sregistry credentials at $HOME/.sregistry you shouldn't need to specify it again:

```bash
$ sregistry push --name container/repo /path/to/container.sif
```

**Comment by sgabenov** [comment]
Thank  you so much!

**Comment by vsoch** [comment]
Let me know if you have trouble. If you had trouble finding the page for push, could you show me where (in the current sregistry docs) you would have expected it to be? We can add it there.

**Comment by sgabenov** [comment]
I think, that it would be great if you can add to the https://singularityhub.github.io/sregistry/client paragraph on how to add custom sregistry hub into user profile. Actually it was not obvious to me that a connection string can be found at sregistry>user>tokens
![image](https://user-images.githubusercontent.com/15741789/55142642-29accc00-514e-11e9-9de5-43a909f5c02e.png)

And also add notes that this string should be added to $HOME/.sregistry and how to make a push into this new configured sregistry (The commands you have helped me with).
Thank you once again for help!

**Comment by vsoch** [comment]
I'll add another link to the [credentials](https://singularityhub.github.io/sregistry/credentials) page there - it does walk you through finding and exporting the token.

**Comment by vsoch** [comment]
This will be closed with #186
